frontend/src/queries/nodes/HogQLQuery/HogQLQueryEditor.tsx

```diff
@@ -241,3 +250,36 @@ export function HogQLQueryEditor(props: HogQLQueryEditorProps): JSX.Element {
         </div>
     )
 }
+
+// one off component for query editor tabs
+interface QueryTabProps {
+    model: Uri
+    active?: boolean
+    onClick?: (model: Uri) => void
+    onClear?: (model: Uri) => void
+}
+
+function QueryTab({ model, active, onClear, onClick }: QueryTabProps): JSX.Element {
+    return (
+        <button
```

Comment:
Should we use LemonTabs or something similar for this?

Reply:
avoided lemontabs as I wanted custom UI that was very much editor-tab like

Reply:
just updated the images in the PR description to match